pimd/pim_cmd.c

```diff
@@ -2780,33 +2858,32 @@ DEFUN (show_ip_pim_cand_rp,
 	return CMD_SUCCESS;
 }
 
-DEFUN (show_ip_pim_bsr_rpdb,
```

Comment:
another command just removed?

Reply:
This is the only command that existed before the new BSR as part of this PR. I made  `show_ip_pim_bsr_rpdb` a hidden command, I didn't remove it. Since now we have multiple bits of BSR info, I chose to move all show commands under a common command `show ip pim bsr...`, just for consistency and better user interface.

Reply:
To clarify further, we now have 
```
show ip pim bsr rp-info
show ip pim bsr candidate-rp
show ip pim bsr groups
show ip pim bsr
```
and more...